### PR TITLE
add examples for common processes.

### PR DESCRIPTION
**Pull request by phoolish**

### CODE REVIEW

File: examples/sidekiq.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+processes:
+  - name: sidekiq
+    config:
+      start_command: bundle exec sidekiq --config config/sidekiq.yml --daemon --logfile log/sidekiq.log
+      stop_command: bundle exec sidekiqctl shutdown tmp/pids/sidekiq.pid
+      user_commands:
+        rotate: "kill -USR2 {PID}" # Requires eye >= 0.6.4
+      stdall: log/sidekiq.log
+      pid_file: tmp/pids/sidekiq.pid
```
Comment (rdunlop) [inline examples/sidekiq.yml]: what about:
```ruby
   notify:
        crash: error
```

not needed? (it's in GradeSaver stage)

Comment (phoolish) [inline examples/sidekiq.yml]: I think notification examples should be separate.

File: examples/unicorn.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Unicorn process example with RAILS_ENV variable
+
+processes:
+  - name: unicorn
+    config:
+      pid_file: tmp/pids/unicorn.pid
+      start_command: bundle exec unicorn -c config/unicorn/<%= ENV['RAILS_ENV'] %>.rb -D
```
Comment (ahorner) [inline examples/unicorn.yml]: We can put the flags together to improve readability a bit:
```yaml
start_command: bundle exec unicorn -Dc config/unicorn/<%= ENV["RAILS_ENV" %>.rb
```

Realistically, this file should be given an `.erb` extension to make it obvious that we're using ERB features here.

File: examples/unicorn.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Unicorn process example with RAILS_ENV variable
+
+processes:
+  - name: unicorn
+    config:
+      pid_file: tmp/pids/unicorn.pid
+      start_command: bundle exec unicorn -Dc config/unicorn/production.rb
```
Comment (phoolish) [inline examples/unicorn.yml]: @ahorner I think the ENV reference should either be a separate example or a README update.

Comment (ahorner) [inline examples/unicorn.yml]: I'm completely okay with that.

**Pull request merged**